swagger.yaml

```diff
@@ -978,6 +978,24 @@
           },
           "styles" : {
             "$ref" : "#/components/schemas/BuilderContentStyles"
+          },
+          "labelPreview": {
+            "type": "string",
+            "description": "Text displayed in the launcher element. If not set the text container is not rendered."
+          },
+          "imgUrl": {
+            "type": "string",
+            "description": "URL for custom image in launcher, overrides the icon settings"
+          },
+          "imageBorderColor": {
+            "type": "string",
+            "description": "CSS color to be used for a border around the custom image in launcher",
+            "default": "If not set will user background color set in the configuration"
```

Comment:
Let's use the brand color instead, the background color usually is plain white or something equally discrete. Also typo in use(r) ;-)